### ISSUE TITLE
No Error's, but no Notification

### ISSUE THREAD

**Issue by git-eri** [Berghmans/cmk_discord#1]
I am really interested in getting some notifications on Discord but i just can't make it happen.

I did everything the readme.md told me to do so and rechecked everything. I think it may be a checkmk installation failure on my end but maybe someone could help me find the right settings.

The output of ~/var/log/notify.log
```
2024-12-15 22:54:12,399 [20] [cmk.base.notify] Analysing notification (10.1.1.1) context with 39 variables
2024-12-15 22:54:12,399 [20] [cmk.base.notify] Global rule 'Discord'...
2024-12-15 22:54:12,399 [20] [cmk.base.notify]  -> matches!
2024-12-15 22:54:12,399 [20] [cmk.base.notify]    - adding notification of automation via discord.py
2024-12-15 22:54:12,399 [20] [cmk.base.notify] Executing 1 notifications:
2024-12-15 22:54:12,399 [20] [cmk.base.notify]   * would notify automation via discord.py, parameters: https://discord.com/api/webhooks/*******3/*******5, bulk: no
```
This is when i try to test the notification. 1 notification rules are matching and 1 resulting notifications for the automation user.
But no notification in the Discord channel.
Any ideas what i could have misconfigured? Is there another logfile i could look for errors?
Already checked dns and firewall, everything is reachable.
Checkmk Raw Edition 2.3.0p21

**Comment by Cillu** [comment]
This happened to me as well and then I realized that sending out a test notification doesn't seem to work. Instead, what got this working was to:

1. Within the notification configuration page, click on 'Show analysis'
2. Click 'replay this notification' on any of your previously sent notifications
3. Boom, notifications!

**Comment by git-eri** [comment]
@Cillu Yes this is what i found out a bit later too, testing the notification provider does not seem to work properly, but real notifications do actually work!

**Comment by OBerghmans** [comment]
For what's it worth, you can (now) also check the "Trigger notification for a specific method" checkbox to really test it with the notifications:

<img width="524" height="223" alt="Image" src="https://github.com/user-attachments/assets/37d6d499-ef8e-402a-86ae-2140619f29ed" />

I'll be closing this but feel free to create a new issue if something is not working
